src/components/Main.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+
+import Dashboard from './Dashboard/Dashboard';
+import BookDetail from './BookDetail/BookDetail';
+import './Main.css';
+
+function Main() {
+  return (
+    <main className="main">
+      <Switch>
```

Comment:
I would do this switch directly in the app component to avoid having more nested components

Reply:
noted, I'd rather not change it because I have like 5 PRs ahead of this one and I don't want to cry while I rebase for this training